.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -39,24 +45,83 @@ If any further question about AME onboarding or validation tools, please view th
+If you have any breaking changes as defined in the [Breaking Change Policy](http://aka.ms/bcforapi), request approval from the Breaking Change Review Board.
+  
+**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Additional details on the process and office hours are on the [Breaking Change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).
+  
+NOTE: To update API(s) in public preview for over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))
 
 Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).
+  
+</details>
+
+## Data Plane API
+
+<details>
+  <summary>Expand this section for dataplane</summary>
+
+This PR template is for Data Plane APIs only.
+
+## API Info: The Basics
+Most of the information about your service should be captured in the issue that serves as your [*engagement record*](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/271/Azure-REST-API-Stewardship?anchor=rest-api-stewardship-process).
+
+* Link to engagement record issue: 
+
+Is this review for (select one):
+
+- [ ] a private preview
+- [ ] a public preview
+- [ ] GA release 
+
+### What are you changing?
+<sup>This section will help us focus on the specific parts of your API that are new or have been modified. <br/>Please share a link to the design document for the new APIs, a link to the previous Open API document (swagger) if applicable, and the root paths that have been updated. </sup>
+
+#### Description
+< Your text here >
+  
+#### References & Links  
+* Design Document: 
+* Previous Open API Doc: 
+* Updated paths:
+  
+## :grey_question: Got questions? Need additional info?? We are here to help!
+
+<details>
+  <summary> Contact us!</summary>
+
+The [Azure API Review Board](https://aka.ms/azapi) is dedicated to helping you create amazing APIs. You can read about our mission and learn more about our process on our [wiki](https://aka.ms/azapi).
+* :speech_balloon: [Teams Channel](https://teams.microsoft.com/l/channel/19%3a3ebb18fded0e47938f998e196a52952f%40thread.tacv2/General?groupId=1a10b50c-e870-4fe0-8483-bf5542a8d2d8&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47)
+* :love_letter: [email](mailto://azureapirbcore@microsoft.com)
+
+</details>
+
+<details>
+  <summary>Click here for links to tools, specs, guidelines & other good stuff</summary>
+  
+### Tooling
+ * [Open API validation tools](https://aka.ms/swaggertools) were run on this PR. Go here to see [how to fix errors](https://aka.ms/ci-fix)
+ * [Spectral Linting](https://aka.ms/style)
+ * [Open API Hub](https://aka.ms/openapiportal)
+
+### Guidelines & Specifications
+ * [Azure REST API Guidelines](https://aka.ms/guidelines)
```

Comment:
@czubair have you validated all these links? I haven't check them all but at least this one doesn't work.